code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -18,7 +18,7 @@
 /mob/living/carbon/human/bee_friendly()
 	if(dna && dna.species && dna.species.id == "pod") //bees pollinate plants, duh.
 		return 1
-	if (wear_suit && head && is_type_in_typecache(wear_suit, GLOB.typecache_clothing) && is_type_in_typecache(wear_suit, GLOB.typecache_clothing))
+	if (wear_suit && head && istype(wear_suit, /obj/item/clothing) && istype(wear_suit, /obj/item/clothing))
```

Comment:
Seems like an `isclothing` may be in order.

Reply:
why are there two checks for wear_suit being clothing here? im pretty sure its redundant

Reply:
Copypaste bug, it's been fixed elsewhere but not here, the second one should be checking `head`.